src/zjs_ashell.json

```diff
@@ -14,9 +14,10 @@
         "sensor_gyro",
         "sensor_light",
         "sensor_temp",
-        "spi"
+        "spi",
+        "fs"
```

Comment:
Does it still fit on A101? That's why we pared things down before.

Also, please put this in alphabetical order.

Reply:
Size is still Ok. FS was already included in ashell build, so no change in the size.